src/features/metrics/aggregate/index.js

```diff
@@ -23,19 +30,11 @@ export class Aggregate extends AggregateBase {
     this.singleChecks() // checks that are run only one time, at script load
     this.eachSessionChecks() // the start of every time user engages with page
 
-    var harvestTimeSeconds = getConfigurationValue(this.agentIdentifier, 'metrics.harvestTimeSeconds') || 30
-
-    var scheduler = new HarvestScheduler('jserrors', { onUnload: () => this.unload() }, this)
+    // *cli, Mar 23 - Per NR-94597, this feature should only harvest ONCE at the (potential) EoL time of the page.
```

Comment:
```suggestion
    // We are no longer harvesting periodically via `scheduler.startTimer` because per NR-94597 in March 2023, this
    // feature should only harvest ONCE at the (potential) EoL time of the page. (EoL collection is the default
    // behavior of HarvestScheduler even if `startTimer` is not invoked.)
```

Reply:
I think comments should serve to remind what the expected behavior is (readers can dig into historic info themselves) rather than cluttering current code w/ _how it was_ b/c they don't immediately see that change and just gets confused.